usaspending_api/awards/tests/test_awards_idv_v2.py

```diff
@@ -274,6 +402,13 @@ def test_award_endpoint_different_ids(client, awards_and_transactions):
     assert json.loads(resp.content.decode("utf-8")) == expected_response_idv
 
 
+@pytest.mark.django_db
+def test_award_endpoint_for_null_recipient_information(client, awards_and_transactions):
+    resp = client.get("/api/v2/awards/3/", content_type="application/json")
+    assert resp.status_code == status.HTTP_200_OK
+    assert json.loads(resp.content.decode("utf-8")) == expected_response_idv_without_recipient_id_and_name
```

Comment:
Very good choice making this a variable, but it looks like a lot of that object is filler. Could we do the asserts on specific sub-objects that are important to the logic in order to make the test clearer? It would also allow us to make the objects smaller and easier to modify for new tests.
```suggestion
    assert json.loads(resp.content.decode("utf-8")).recipient == expected_response_idv_without_recipient_id_and_name.recipient
```
(note: I didn't test this suggestion. Just wanted to show an example)